framework/src/node/state_machine/state_machine.ts

```diff
@@ -125,7 +125,7 @@ export class StateMachine {
 			}
 			return { status: VerifyStatus.OK };
 		} catch (error) {
-			return { status: VerifyStatus.FAIL };
+			return { status: VerifyStatus.FAIL, error: error as Error };
 		}
```

Comment:
Maybe,
```ts
} catch (error: Error) {
			return { status: VerifyStatus.FAIL, error };
		}
```

Reply:
`} catch (error: Error) {` doesn't work since typescript type it as any